bcipy/tasks/start_task.py

```diff
@@ -44,10 +43,6 @@ def make_task(display_window, daq, exp_type, parameters, file_save,
             display_window, daq, parameters, file_save, signal_model,
             language_model, fake=fake)
 
-    if exp_type is ExperimentType.RSVP_COPY_PHRASE_CALIBRATION:
-        return RSVPCopyPhraseCalibrationTask(
-            display_window, daq, parameters, file_save, fake)
-
```

Comment:
I wasn't sure if this was ever used for anything. Nice to clean these things up.